script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -412,12 +412,12 @@ From the long―― unnaturally long―― shadow that stretches before him emer
 [sha:b96d31336608a041ff2c5505466202aaa047b946]{
 -- Page 545, Offset 10061.
 -- 「だが、ご当主にとっても、ソレはより不相応だった。
-"However, she was utterly undeserving of the position of family head.
+"However... my liege was not suited of it either.
```

Comment:
suited _to_ it

Reply:
"suited of it" has no ngram results even if i set it to go back as far in time as it'll go

Reply:
thinking emoji